### PR TITLE
📝 Add macOS-specific instructions for local development.

### PR DESCRIPTION
**Pull request by jcfrancisco**
👋

Here are steps that worked to set up the app on my macOS machine. Let me know if I should change any wording!

Just in case this isn't a known issue: I couldn't run `cargo build`. It seems like two packages can't be retrieved because their git repo is offline.

```
Updating git repository `https://git.asonix.dog/asonix/activitystreams-ext`
warning: spurious network error (2 tries remaining): failed to connect to git.asonix.dog: Operation timed out; class=Os (2)
```

### CODE REVIEW

File: docs/src/contributing_local_development.md [hunk 1]
```diff
@@ -1,16 +1,26 @@
-### Ubuntu
-
-
-#### Build requirements:
+### Install build requirements
+#### Ubuntu
 ```
-sudo apt install git cargo libssl-dev pkg-config libpq-dev yarn curl gnupg2 git
+sudo apt install git cargo libssl-dev pkg-config libpq-dev yarn curl gnupg2
```
Comment (jcfrancisco) [inline docs/src/contributing_local_development.md:4]: Assuming this extra `git` was left in here by accident, but lmk otherwise.

Comment (Nutomic) [reply docs/src/contributing_local_development.md:4]: Thanks, good catch!

**Comment by Nutomic** [comment]
Yes that build error is expected, @asonix is moving so their home server is currently down. Probably best to wait until its back to make sure everything works.

**Comment by dessalines** [comment]
Looks good to me, thanks!

**Pull request merged**